extensions/linz/README.md

```diff
@@ -24,10 +24,15 @@ extension which adds constraints to default STAC schema properties.
 
 ## Item Fields
 
-| Field Name           | Type                | Description                                                                                                                                                  |
-| -------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| linz:geospatial_type | string              | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types). |
-| processing:software  | Map<string, string> | Recommended. The software and versions which were used to generate the dataset. See [reference](https://github.com/stac-extensions/processing).              |
+| Field Name          | Type                | Description                                                                                                                                     |
+| ------------------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| processing:software | Map<string, string> | Recommended. The software and versions which were used to generate the dataset. See [reference](https://github.com/stac-extensions/processing). |
```

Comment:
I think this is also in the properties object isn't it?
https://github.com/stac-extensions/processing#item-properties-and-collection-provider-fields